### PR TITLE
MapUtil - Add util filtering layers by property

### PR DESCRIPTION
**Pull request by ahennr**
### Description:

This PR adds a further util returning the list of layers matching the given pair of properties.

### CODE REVIEW

**Comment by annarieger** [comment]
Nice addition, please merge at will :pineapple:

File: src/Util/MapUtil/MapUtil.js [hunk 1]
```diff
@@ -244,6 +244,23 @@ export class MapUtil {
...
   }
 
+  /**
+   * Returns the list of layers matching the given pair of properties.
+   *
+   * @param {ol.Map} map The map to use for lookup.
+   * @param {String} key The property key.
+   * @param {Object} value The property value.
+   *
+   * @return {ol.layer.Base} The list of matching layers.
```
Comment (dnlkoch) [inline src/Util/MapUtil/MapUtil.js]: `{ol.layer.Base}` -> `{ol.layer.Base[]}`

Comment (ahennr) [reply src/Util/MapUtil/MapUtil.js]: Fixed in 52896b2ce70e

**Comment by ahennr** [comment]
Thx for review @annarieger and @dnlkoch

**Pull request merged**